docs-standards/10-structure-the-content/01-document-types/quickstart-help-me-get-started.md

```diff
@@ -0,0 +1,182 @@
+# Quickstart - Help me get started
+
+> ℹ️ **Note:**
+>
+> For the template associated to this topic, see [quickstart template](../../../templates/quickstart/quickstart-template.md).
+
+A quickstart focuses on the primary aspects of the product or feature, helping your users to start using your product as quickly as possible. It reduces users' onboarding time and gives them the feeling that the product is easy to use.
+
+You need at least one quickstart for each product. If the product is complex, you can create multiple quickstarts for different features or tasks.
+
+## Write a quickstart
+
+### Filenames
+
+Check out the [file names and location rules](../../../CONTRIBUTING.md#file-names-and-location-rules) for more information.
+
+### General rules
+
+- The entire quickstart should be about 1 000 words long.
```

Comment:
The quickstart guides created by GitHub and Mircrosoft are about 600 - 1000 words long, so I put 1000 here